.travis.yml

```diff
@@ -2,11 +2,19 @@ language: node_js
 dist: trusty
 sudo: required
 node_js: stable
+env:
+  - SIMPLE_STORAGE_ADDRESS: "0x7ec89063be181f1ad3bcea675bb6413da0f3f50b"
 install:
-  - npm install -g bower
```

Comment:
Why remove it from here?

Reply:
I think it's ok to expect bower being installed globally (which it usually is, at least for purescript devs).

Reply:
I added `bower` to devDeps so this is no longer necessary. It once bit me on a psc-package project having an implicit dependency on bower so I just like adding it.